src/renderers/shaders/UniformsLib.js

```diff
@@ -18,6 +18,7 @@ var UniformsLib = {
 		offsetRepeat: { value: new Vector4( 0, 0, 1, 1 ) },
 
 		specularMap: { value: null },
+		uSpecularStrength: { value : 1 },
```

Comment:
We currently have the `reflectivity` property that modulates the specular map. There is no reason to add another property that does the same thing.

The specular map is going to become RGB-valued for Phong. A shininess map will be added, also. Optionally, the reflectivity property will modulate a reflectivity map, if we chose to add it.